src/forklift/cli.py

```diff
@@ -413,3 +415,14 @@ def update_static(file_path):
     log.info('%s', report)
 
     return report
+
+
+def scorched_earth():
+    staging = config.get_config_prop('stagingDestination')
+    for folder in [staging, core.hash_gdb_path, core.scratch_gdb_path]:
+        if exists(folder):
+            log.info('deleting: %s', folder)
+            rmtree(folder)
```

Comment:
we might want to try catch this or do we really care? The message from python will be descriptive already...